dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/JdbcEventStore.java

```diff
@@ -2074,6 +2074,21 @@ public void updateTrackedEntityInstances( List<String> teiUids, User user )
             PreparedStatement::execute );
     }
 
+    @Override
+    public void updateEnrollmentsLastUpdatedUserInfo( List<String> enrollmentUids, UserInfoSnapshot userInfoSnapshot )
```

Comment:
Does this only update `enrollments`? If so I would expect this to be in an  `EnrollmentStore` rather then `EventStore` 🤔 Unless that leads us to a cyclic dependency between enrollments <-> events. I am bit worried about the lack of structure when it comes to our module and package dependencies. That's why I am bringing this up.